src/genericSchemeRegistry/schemes/BalancerPoolManager.json

```diff
@@ -0,0 +1,341 @@
+      "notes": "https://github.com/PrimeDAO/monorepo/blob/master/packages/contracts/contracts/schemes/BalancerProxy.sol",
+      "fields": [
+        {
+          "label": "Token address",
+          "name": "token",
+          "placeholder": "Address (0x0000…)"
+        },
+        {
+          "decimals": 18,
+          "label": "New weight of the Token.",
+          "name": "new weight",
+          "unit": "unit",
+          "placeholder": "Weight (1500)"
+        }
+      ],
+      "abi": {
+          "constant": false,
+          "inputs": [
+            {
+              "internalType": "address",
+              "name": "token",
+              "type": "address"
+            },
+            {
+              "internalType": "uint256",
+              "name": "newWeight",
+              "type": "uint256"
+            }
+          ],
+          "name": "updateWeight",
+          "outputs": [],
+          "payable": false,
+          "stateMutability": "nonpayable",
+          "type": "function"
+      }
+    },
+    {
+      "id": "updateWeightsGradually",
+      "label": "Update token weights",
+      "description": "Sets token weights (to be gradually updated)",
+      "notes": "https://github.com/PrimeDAO/monorepo/blob/master/packages/contracts/contracts/schemes/BalancerProxy.sol",
+      "fields": [
+        {
+          "label": "New weights",
+          "name": "New weights that are going to be gradually updated",
+          "placeholder": "Weight (10)"
+        },
+        {
+          "decimals": 18,
+          "label": "Start block",
+          "name": "Start update from block",
+          "unit": "unit",
+          "placeholder": "Weight (199199)"
+        },
+        {
+          "decimals": 18,
+          "label": "End block",
+          "name": "End update at block",
+          "unit": "unit",
+          "placeholder": "Weight (399199)"
+        }
+      ],
+      "abi": {
+      "constant": false,
+      "inputs": [
+              {
+                "internalType": "uint256[]",
+                "name": "newWeights",
+                "type": "uint256[]"
+              },
+              {
+                "internalType": "uint256",
+                "name": "startBlock",
+                "type": "uint256"
+              },
+              {
+                "internalType": "uint256",
+                "name": "endBlock",
+                "type": "uint256"
+              }
+            ],
+            "name": "updateWeightsGradually",
+            "outputs": [],
+            "payable": false,
+            "stateMutability": "nonpayable",
+            "type": "function"
+            }
+    },
+    {
+      "id": "joinPool",
+      "label": "Join pool",
+      "description": "Joins the pool by adding more tokens",
+      "notes": "https://github.com/PrimeDAO/monorepo/blob/master/packages/contracts/contracts/schemes/BalancerProxy.sol",
+      "fields": [
+        {
+          "decimals": 18,
+          "label": "Pool Amount Out",
```

Comment:
"Pool amount out"  Why is this in wei? Seems odd if this is just a number of tokens, especially when the second param  has not decimals.